### DISCUSSION TITLE
Inverting the IR output with no carrier.

### DISCUSSION THREAD

**Discussion by AJLelec** [Arduino-IRremote/Arduino-IRremote#1229] [Help] [answered]
I have been trying to get a piece of equipment to respond for some time via a wired connection.  I was really pleased to see the option in version 4 to turn off the carrier and I have successfully used that to control another similar device from another manufacturer which uses Active low with no carrier.  

I managed to find a different, commercial controller for this equipment and I have ascertained that the signal it requires is NEC protocol, no carrier and Active HIGH, rather than active low.  I was unable to find any method of inverting the output and so if it exists, I would love to be pointed at it please?

Despite much searching, I am unable to find any mention of inverting the output anywhere in the documentation, I promise that I have spent a long time looking!

**Comment by ArminJo** [comment, marked as answer]
This was a good and clear request 👍 
So it is implemented now. Please test it.
Just use:
`#define USE_ACTIVE_HIGH_OUTPUT_FOR_NO_SEND_PWM // Simulate an active high receiver signal instead of an active low signal.`

**Reply by AJLelec** [reply]
I am most grateful to you, it is perfect. Thanks again.
